runtime/proto/service/corfu_message.proto

```diff
@@ -0,0 +1,159 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime.proto.service";
+
+import "common.proto";
+import "service/base.proto";
+import "service/layout.proto";
+import "service/log_unit.proto";
+import "service/sequencer.proto";
+import "service/management.proto";
+import "server_errors.proto";
+
+//TODO: Reasonable way to determine version?
+enum ProtocolVersion {
```

Comment:
Instead of an enum can this be a regular protobuf that includes a fixed64?
Version of V0 by itself does not have much use, but if this field can be expanded, we can potentially consider this to be our version capability exchange. since this is included into all the headers.

Reply:
Done. It remains unused for the time being, but this could be a good approach.